app/views/shared/_alerts.html.erb

```diff
@@ -4,6 +4,9 @@
       <div class="alert alert-<%= klass %>">
         <button type="button" class="close" data-dismiss="alert">&times;</button>
         <%= message %>
+        <% if display_access_request_link?(type) %>
+          <%= link_to_request_access %>
+        <% end %>
```

Comment:
If we could do something like what's described in here: http://stackoverflow.com/questions/26538891/flash-message-with-html-safe-from-the-controller-in-rails-4-safe-version, we could benefit from it a lot in this code snippet, as we could keep it completely generic, i.e., agnostic from the errors being displayed.

Still, personally, I don't like to see html being built on the controller, so, is there another, better, way to accomplish this same behaviour?